client/features/lib/styl/features.styl

```diff
@@ -119,7 +119,7 @@ body.features
         h5
           font-size           21px
           line-height         22px
-          color               #27ae60
+          color               #48BA7D
```

Comment:
Leaving it as is since all the greens updated, instead of adding one more commit.